tests/detect-flowbits/test.yaml

```diff
@@ -0,0 +1,80 @@
+requires:
+  min-version: 6
+
+args:
+- -k none
+
+checks:
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 1
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 2
+- filter:
+    count: 0
+    match:
+      event_type: alert
+      alert.signature_id: 3
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 5
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 6
+- filter:
+    count: 0
+    match:
+      event_type: alert
+      alert.signature_id: 7
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 8
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 9
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 10
+- filter:
+    count: 0
+    match:
+      event_type: alert
+      alert.signature_id: 11
+- filter:
+    count: 0
+    match:
+      event_type: alert
+      alert.signature_id: 12
+- filter:
+    count: 0
+    match:
+      event_type: alert
+      alert.signature_id: 13
+- filter:
+    count: 0
```

Comment:
This does not seem right. We set flowbits `fb1` and `fb2` in `sid`s 1 and 2, so both are set and were never unset them in the following rules. This rule should alert.

Reply:
I'm not sure why it does not alert.